Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -84,6 +84,22 @@ void OnDestroy()
             }
         }
 
+        void OnDisable()
+        {
+            if (keywordRecognizer != null)
+            {
+                StopKeywordRecognizer();
+            }
+        }
+
+        void OnEnable()
+        {
+            if (keywordRecognizer != null && RecognizerStart == RecognizerStartBehavior.AutoStart)
```

Comment:
When app starts with KeywordManager disabled and is enabled while executing, keywordRecognizer will be null so StartKeywordRecognizer won't be called. 
StartKeywordRecognizer already verifies if keywordRecognizer is null so this verification can be removed from here.
This condition also means that, if start behavior is not AutoStart, when KeywordManager is enabled it won't start. Is this the expected behavior?